nrf_modem/Kconfig

```diff
@@ -33,6 +33,10 @@ config NRF_MODEM_LINK_BINARY_DECT_PHY
 	select EXPERIMENTAL
 	bool "DECT PHY"
 
+config NRF_MODEM_LINK_BINARY_DECT_NRP
+	select EXPERIMENTAL
+	bool "DECT NR+"
```

Comment:
Also, as said in Teams, I'd like to have a main flag for the DECT, and maybe also for the cellular.
I.e. to get rid of using CONFIG_NRF_MODEM_LIB as a synonym for LTE/cellular related code. See ncs usage for that and you'll see the point.

Reply:
I think that it is must to be solved somehow before these are going to some actual ncs release. Before this PR, this wasn't a problem in practice, as there were just cellular and dect_phy (that was not used for example in IP level or for cloud access).